pgml-dashboard/src/components/headings/gray/gray.scss

```diff
@@ -0,0 +1,3 @@
+span[data-controller="headings-gray"] {
+    color: #{$gray-400};
+}
```

Comment:
You could add color = gray-400 to the typography.scss file and just use the class here.  Utility classes for the gray scale is always nice.